composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.6.0",
         "illuminate/support": "5.2.* || 5.4.* || 5.5.* || 5.6.*|| 5.7.*|| 5.8.*|| 6.0.* || ^6 || ^7",
-        "ramsey/uuid": "^3.2"
+        "ramsey/uuid": "^4.0"
```

Comment:
I think we might be able to make this `"ramsey/uuid": "^3.2 || ^4.0"` and support both. I'm guessing this will work

Reply:
You're right @jamesmills , thank you.

Reply:
Update done, thank you @jamesmills